make/Make.demos.rules

```diff
@@ -27,7 +27,7 @@ demo-programs           = client server publisher subscriber reader writer
 # The default demo sources for each demo executable.
 #
 demo-client-sources     = Client.$1 *.ice
-demo-server-sources     = Server.$1 *.ice *I.$1
+demo-server-sources     = Server.$1 *.ice *I.$1 Chatbot.$1
```

Comment:
I am not a fan of this code.

It should be easy for the user to figure out which files are compiled into which executable and currently it's all magic.

Reply:
I think ideally we would just have a simple Makefile per demo with a little more "duplication".